formats/polus-ome-zarr-converter-plugin/plugin.json

```diff
@@ -1,45 +1,57 @@
+
 {
-  "name": "OME Zarr Converter",
-  "version": "0.2.0",
-  "title": "OME Zarr Converter",
-  "description": "Convert Bioformats supported format to OME Zarr.",
-  "author": "Nick Schaub (nick.schaub@nih.gov)",
-  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
-  "repository": "https://github.com/labshare/polus-plugins",
-  "website": "https://ncats.nih.gov/preclinical/core/informatics",
-  "citation": "",
-  "containerId": "labshare/polus-ome-zarr-converter-plugin:0.2.0",
-  "inputs": [
-    {
-      "name": "inpDir",
-      "type": "genericData",
-      "description": "Input generic data collection to be processed by this plugin",
-      "required": true
-    },
-    {
-      "name": "filePatter",
-      "type": "string",
-      "description": "A filepattern, used to select data to be converted",
-      "required": true
-    }
-  ],
-  "outputs": [
-    {
-      "name": "outDir",
-      "type": "genericData",
-      "description": "Output collection"
-    }
-  ],
-  "ui": [
-    {
-      "key": "inputs.inpDir",
-      "title": "Input generic collection",
-      "description": "Input generic data collection to be processed by this plugin"
-    },
-    {
-      "key": "inputs.filePattern",
-      "title": "Filepattern",
-      "description": "A filepattern, used to select data for conversion"
-    }
-  ]
-}
\ No newline at end of file
+    "name": "OME Zarr Converter",
+    "version": "0.5.0",
+    "title": "OME Zarr Converter",
+    "description": "Convert Bioformats supported format to OME Zarr.",
+    "author": "Nick Schaub (nick.schaub@nih.gov)",
+    "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+    "repository": "https://github.com/labshare/polus-plugins",
+    "website": "https://ncats.nih.gov/preclinical/core/informatics",
+    "citation": "",
+    "containerId": "labshare/polus-ome-zarr-converter-plugin:0.5.0",
+    "inputs": [
+      {
+        "name": "inpDir",
+        "type": "genericData",
+        "description": "Input generic data collection to be processed by this plugin",
+        "required": true
+      },
+      {
+        "name": "filePattern",
+        "type": "string",
+        "description": "A filepattern, used to select data to be converted",
+        "required": true
+      },
+      {
+        "name": "fileExtension",
+        "type": "string",
+        "description": "Type of data conversion",
+        "required": true
```

Comment:
This should not be a required argument, and should be an enum. We only support two standard file formats, so let's not make it easy for a user to select something that we don't support.

Reply:
I change the type to enum and provided user to select from the two options only
 ".ome.tif",
 ".ome.zarr"